pkg/network/port_test.go

```diff
@@ -120,17 +121,35 @@ func TestReadInitialUDPState(t *testing.T) {
 	err := exec.Command("testdata/setup_netns.sh", nsName).Run()
 	require.NoError(t, err, "setup_netns.sh failed")
 
-	l, err := net.ListenUDP("udp", &net.UDPAddr{})
-	require.NoError(t, err)
-	defer func() { _ = l.Close() }()
+	l := nettestutil.StartServerUDP(t, net.ParseIP("0.0.0.0"), 0)
+	t.Cleanup(func() {
+		defer l.Close()
+	})
 
-	l6, err := net.ListenUDP("udp6", &net.UDPAddr{})
-	require.NoError(t, err)
-	defer func() { _ = l.Close() }()
+	l6 := nettestutil.StartServerUDP(t, net.ParseIP("::"), 0)
+	t.Cleanup(func() {
+		defer l6.Close()
+	})
+
+	conn, ok := l.(*net.UDPConn)
+	assert.True(t, ok)
```

Comment:
nit: you can use require here. If this assertion fails, then `conn` will be `nil` and line 139 will panic.